console/console-init/ui/src/modules/connection/components/ConnectionToolbar/ConnectionToolbar.tsx

```diff
@@ -4,6 +4,7 @@
  */
 
 import * as React from "react";
```

Comment:
remove * as we don't need to import whole react module. we need default exports only.
import React from "react";